src/dbnode/server/server.go

```diff
@@ -183,6 +194,26 @@ func Run(runOpts RunOptions) {
 		logger.Fatalf("could not resolve local host ID: %v", err)
 	}
 
+	var (
+		tracer      opentracing.Tracer
+		traceCloser io.Closer
+	)
+
+	if cfg.Tracing == nil {
+		tracer = opentracing.NoopTracer{}
+		logger.Infof("tracing disabled for %s; set `tracing.backend` to enable", serviceName)
+	} else {
+		// setup tracer
+		tracer, traceCloser, err = cfg.Tracing.NewTracer(serviceName, scope, zapLogger)
```

Comment:
hm should this be given a subscope to keep metrics namespaces clean? (not sure if jaeger's instrumentation is doing so)

Reply:
Looks like it just wraps it. want me to add a `.SubScope("jaeger")`?
```
// Wrap takes a tally Scope and returns jaeger-lib metrics.Factory.
func Wrap(scope tally.Scope) metrics.Factory {
	return &factory{
		tally: scope,
	}
}
```

Reply:
yep sounds good